profondo/cnn.py

```diff
@@ -28,26 +29,35 @@ def load_data(path):
     return meta, images
 
 
-def train_test_split(meta, images, train_ratio=0.8):
+def train_test_split(meta, images, train_ratio=0.8, validation_ratio=0.2):
     idx = np.random.permutation(np.arange(len(images)))
     train_size = int(len(idx)*train_ratio)
-    train_idx = idx[:train_size]
-    test_idx = idx[train_size:]
+    validation_size = int(train_size*validation_ratio)
```

Comment:
I think the train ratio should be kept as specified. The user can then specify the train and validation ratio. The test ratio will be `1 - train_ratio - validation_ratio`.